### PR TITLE
Update FAQ/SUPPORT - reverse proxy is supported

### PR DESCRIPTION
**Pull request by wxwr**

### CODE REVIEW

**Review by yvonnevictoria — APPROVED** [review]

**Comment by yvonnevictoria** [comment]
Thanks @wxwr! :)

File: docs/FAQs.md [hunk 1]
```diff
@@ -57,8 +57,6 @@ When an organization contains a small amount of data, the backfilling process is
 
 - **Atlassian IP address ranges need whitelisting** - [Please see our IP allowlist documentation](https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md)
 
-- **Reverse proxy support** - GitHub for Jira does not support reverse proxies.
-
```
Comment (atrigueiro) [inline docs/FAQs.md:61]: Should we add a link to https://support.atlassian.com/jira-cloud-administration/docs/connect-a-github-enterprise-server-account-to-jira-software/ here saying that we actually do support? :)

Comment (yvonnevictoria) [reply docs/FAQs.md:61]: Couldn't add a commit to this PR so created a different PR for that here: https://github.com/atlassian/github-for-jira/pull/2631

**Comment by yvonnevictoria** [comment]
On second thought, this should be on DAC. Closing this PR.

**Pull request closed without merging**